prometheus-exporters/vrops-exporter/alerts/datastore.alerts

```diff
@@ -3,13 +3,28 @@ groups:
   rules:
   - alert: DataStoreCapacityWarning
     expr: >
-      max_over_time(vrops_datastore_diskspace_total_usage_gigabytes[10m]) / max_over_time(vrops_datastore_diskspace_capacity_gigabytes[10m]) > 0.9
+      max_over_time(vrops_datastore_diskspace_total_usage_gigabytes[10m]) / max_over_time(vrops_datastore_diskspace_capacity_gigabytes[10m]) >= 0.8 < 9
```

Comment:
FYI `>= 0.8` would work as well (without the `< 9`, which should be **0.9** if you don't omit it)

There is the concept of alert inhibitions: if alerts of the same context are firing (in this case warning >=0.8 and critical >0.9) only the one with the higher severity will create a notification.

Reply:
Thanks for the hint